compiler/src/iree/compiler/Codegen/LLVMGPU/test/transform_dialect_codegen_vector_warp_execute_on_lane_0_spec.mlir

```diff
@@ -0,0 +1,20 @@
+// RUN: iree-opt
+
+transform.with_pdl_patterns {
+^bb0(%arg0: !pdl.operation):
+
+  pdl.pattern @pdl_if_op_target : benefit(1) {
+    %args = operands
+    %results = types
+    %0 = operation "scf.if"(%args : !pdl.range<value>) -> (%results : !pdl.range<type>)
+    
+    // TODO: we don't want this, but it is the required terminator for pdl.pattern
+    rewrite %0 with "transform.dialect"
+  }
+
+  transform.structured.canonicalized_sequence %arg0 {
+  ^bb1(%arg1: !pdl.operation):
+    %if_op = pdl_match @pdl_if_op_target in %arg1
```

Comment:
Not sure why is this test necessary. It only checks that the op with autogenerated syntax can be parsed.

Reply:
old remnant, cleaned.